building/tracks/new/select-programming-language-variant.md

```diff
@@ -0,0 +1,5 @@
+# Select programming language variant
+
+Some programming languages have multiple variants.
+If that is the case, we should decide upon one variant that we'll support.
+Ignore this step if your languages does _not_ have multiple variants.
```

Comment:
The "if" in the first condition guards against any action at all if the condition is not met, the default is to not choose another variant if it does not exist.  It would not even be a question.

```suggestion
```